pkg/util/kubernetes/apiserver/hpa_controller_test.go

```diff
@@ -158,6 +179,7 @@ func TestAutoscalerController(t *testing.T) {
 		"foo",
 		map[string]string{"foo": "bar"},
 	)
+	mockedHPA.Annotations = makeAnnotations("foo", map[string]string{"foo": "bar"})
```

Comment:
what's the goalg of doing this? I don't see additional `assert` in the test

Reply:
It's to be consistent with the modification of the annotation later on. The HPA change is only picked up if the annotation is changed